packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.js

```diff
@@ -1,4 +1,86 @@
 // @vue/component
+import Vue from "vue";
+import SfMegaMenuColumn from "./_internal/SfMegaMenuColumn.vue";
+
+Vue.component("SfMegaMenuColumn", SfMegaMenuColumn);
+
+import SfList from "../SfList/SfList.vue";
+import SfMenuItem from "../../molecules/SfMenuItem/SfMenuItem.vue";
+import SfBar from "../../molecules/SfBar/SfBar.vue";
+
 export default {
-  name: "SfMegaMenu"
+  name: "SfMegaMenu",
+  components: {
+    SfList,
+    SfMenuItem,
+    SfBar
+  },
+  props: {
+    /**
+     * Parent menu
```

Comment:
not very descriptive, what exactly this prop is doing? ^^

Reply:
I split these props to `title` and `visible` now it should be most descriptive :) fix: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/749eec67e11793de3d49be672cfc18808e2584d5